allauth/socialaccount/providers/angellist/provider.py

```diff
@@ -14,7 +14,7 @@ def get_avatar_url(self):
 
     def to_str(self):
         dflt = super(AngelListAccount, self).to_str()
-        return self.account.extra_data.get("name", dflt)
+        return self.account.extra_data.get("email", dflt)
```

Comment:
As above.

Reply:
Same here. The `dflt` value will probably be from the `name` value, since the parent class's `to_str` implementation was changed.